File: backend_StudentskiOnlineServis/DLWMS_StudentskiOnlineServis/Modul_Student/Models/Student.cs
using DLWMS_StudentskiOnlineServis.Modul_1.Models;
using System;

namespace DLWMS_StudentskiOnlineServis.Modul_Student.Models
{
    public class Student : KorisnickiNalog
    {
        public string broj_indeksa { get; set; }
        public string slika_studenta { get; set; }
        public DateTime DatumRodjenja { get; set; }
        public string Ime { get; set; }
        public string Prezime { get; set; }
        public int CijenaSkolarine { get; set; }
    }
}

File: fronted_StudentskiOnlineServis/src/app/StudentModule/_models/Student.ts
import { Fakultet } from "./Fakultet";
import { KorisnickiNalog } from "./KorisnickiNalog";

export interface Student {
  id: number;
  ime: string;
  prezime: string;
  broj_indeksa:string;
  datum_rodjenja: string;
  slika_studenta: string;

  fakultetID: number;
  fakultet: Fakultet;

  korisnickiNalogID: number;
  korisnickiNalog: KorisnickiNalog;

}
